waspc/cli/Command/Telemetry.hs

```diff
@@ -24,7 +24,21 @@ telemetry = do
     telemetryDisabled <- liftIO isTelemetryDisabled
     waspSaysC $ "Telemetry is currently: " <> (if telemetryDisabled
         then "DISABLED"
-        else "ENABLED") 
+        else "ENABLED")
+    
+    -- Code's being duplicated again, need some suggestions to improve this.
+    telemetryCacheDirPath <- liftIO ensureTelemetryCacheDirExists
+    maybeProjectHash <- (Just <$> TlmProject.getWaspProjectPathHash) `catchError` const (return Nothing)
+    let maybeProjectHash = case maybeProjectHash of
+                Nothing          -> Nothing                       -- probably should be reporting an error here 
+                Just projectHash -> TlmProject.readProjectTelemetryFile telemetryCacheDirPath projectHash
+
+    waspSaysC $ TlmProject.showLastSentTelemetryDateTime maybeProjectCache
+
+    -- IDEA: Now that we're reading the content of the file, might as well, show it's content?
```

Comment:
Hmmm maybe! I actually wouldn't do it for now, because I don't think users will get any useful info from it, it might just confuse them (TMI).